source/scholarships.html.slim

```diff
@@ -53,3 +22,5 @@ section.h-lots-of-padding-top
               a href="https://www.rabid.co.nz" rel="noopener"
               .c-sponsor__description
                 p Rabid are a software development consultancy, providing expertise that allows our clients to tackle their big challenges in a digital environment.  We believe that diversity is an important part of any healthy culture, and ensures a range of knowledge and experience can be brought to bear in developing solutions.  We are proud to be providing scholarships to Kiwi Ruby and encouraging the growth of diversity in the Wellington technology culture.
+    .body-inner-wrapper__section
+      p Thanks also to RimuHosting and the Ruby community for donating extra tickets.
```

Comment:
Should I queue up a tweet to thank these ppl?????

Reply:
Sure. One ticket was donated under "RimuHosting", one was donated under "The Ruby Community", and one was donated with no answer to "who should we thank?"